TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,180 @@ A tile is composed of a header section immediately followed by a body section.
 
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
 
 ## Header
 
-The 28-byte header contains the following fields:
+The 36-byte header contains the following fields:
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
+| Field name | Data type | Description |
+| --- | --- | --- |
 | `magic` | 4-byte ANSI string | `"i3dm"`.  This can be used to identify the arraybuffer as an Instanced 3D Model tile. |
 | `version` | `uint32` | The version of the Instanced 3D Model format. It is currently `1`. |
 | `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
-| `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates there is not a batch table. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. |
 | `gltfByteLength` | `uint32` | The length of the glTF field in bytes. |
 | `gltfFormat` | `uint32` | Indicates the format of the glTF field of the body.  `0` indicates it is a url, `1` indicates it is embedded binary glTF.  See the glTF section below. |
-| `instancesLength` | `uint32` | The number of instances, also called features, in the tile. |
 
-_TODO: Link to Cesium code for reading header_
+If either `featureTableJSONByteLength` or `gltfByteLength` equal zero, the tile does not need to be rendered.
 
-If either `gltfByteLength` or `instancesLength` equal zero, the tile does not need to be rendered.
+The body section immediately follows the header section, and is composed of three fields: `Feature Table`, `Batch Table`, and `glTF`.
 
-The body section immediately follows the header section, and is composed of three fields: `Batch Table`, `glTF`, and `instances`.
+Code for reading the header can be found in
+[Instanced3DModelTileContent](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Instanced3DModel3DTileContent.js#L170)
```

Comment:
Change this to:

> [Source/Scene/Instanced3DModelTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Instanced3DModel3DTileContent.js)

If you can give me commit access to your fork, I will make minor changes like this.